### DISCUSSION TITLE
can't show image

### DISCUSSION THREAD

**Discussion by ManarangID** [RTippin/messenger#53] [Q&A] [answered]
hy, I've tried installing Messenger Demo but I'm having trouble changing and sending pictures. what appears is always image404.png from "vendor\messenger\images". also when sending pictures on messenger, where is my mistake?
regards

**Comment by RTippin** [comment]
Can you check your `laravel.log` for errors? Could be a read/write permission in your storage disk/directory. Related config: https://github.com/RTippin/messenger/blob/92cca2804e74b79319be6177083c728ca0244038/config/messenger.php#L44-L53

**Reply by ManarangID** [reply]
this is location link url if open image profil in new tab messenger/assets/provider/user/982c9398-1ef1-4e97-b17a-d8760e28148b/sm/keela_59152754-7ae8-4ce7-a117-2606831465aa.jpg 
![messenger](https://user-images.githubusercontent.com/119088585/211324560-e5608558-ed6f-45b8-b9de-2c65230fd439.png)
![messenger](https://user-images.githubusercontent.com/119088585/211324930-6dd3be51-a30d-4b84-8dc1-9afa91004936.png)

**Comment by RTippin** [comment]
Assuming you left my storage config to its defaults, can you confirm you see the files being stored in storage at least? Please check `storage/app/public/images/users/...` I believe, or just check all of `storage/app`.

**Comment by ManarangID** [comment]
Yes, the image has been successfully uploaded
![messenger](https://user-images.githubusercontent.com/119088585/211493535-324a0495-5c7d-49bb-b5a2-cf17976c263e.png)

**Comment by RTippin** [comment, marked as answer]
Sounds like it could be a permissions/visibility issue then. Check your webserver logs to see if it is blocking access to your storage when serving it.

**Reply by ManarangID** [reply]
ok thanks bro for ur attention and idea
